_po/ja/v1/reference/jsonb.pot

```diff
@@ -37,7 +37,7 @@ msgid ""
 "```"
 msgstr ""
 
-msgid "You can find the JSON by full text search with `search`, `example` or `web` because all text values are full text search target."
+msgid "You can find the JSON by full text search with `server`, `example` or `web` because all text values are full text search target."
```

Comment:
```suggestion
msgid "You can find the JSON by full text search with `server`, `example`, or `web` because all text values are full text search target."
```